src/main/groovy/com/bmuschko/gradle/izpack/CreateInstallerTask.groovy

```diff
@@ -42,7 +42,6 @@ class CreateInstallerTask extends DefaultTask {
     File installFile
 
     @OutputFile
-    @PathSensitive(PathSensitivity.RELATIVE)
```

Comment:
Why did you remove this? It's require for the build cache to work properly. Please revert.

Reply:
Because Gradle 6.x will not allow me. This confirms that I shoudln't have bumped that much. Let's do this again.